### ISSUE TITLE
Update AWS Bucket Policy 

### ISSUE THREAD

**Issue by fasaas** [serverless-components/website#24]
When I deploy using `@serverless/website` I am unable to override the Bucket Policy for the S3 bucket I am creating.

Instead of it being:
```json
"Principal": {
    "AWS": "*"
}
```

I would like it to be configurable to achieve:
```json
"Principal": "*"
```

Is this achievable?
I would say yes since I installed the NPM package locally, and I was able to find the method `configureBucketForHosting` inside `utils.js`, the `Principal` value is hardcoded.

Is there a way for me to create a pull request to @serverless/website?
I am not able to find the open source repo

**Comment by avinasherupaka** [comment]
@fasaas I agree. I would even go a step further and say the bucket policy needs to configurable (some thing passed as an input).
Reason: If you need to protect your bucket or have some custom bucket level policies, the current setup those configurations are overridden after every deployment.

@eahefnawy Thoughts ?
